### ISSUE TITLE
login is not possible with point in user name

### ISSUE THREAD

**Issue by jan-patrick** [OpenbookOrg/openbook-app#108]
After being logged out by the server bug you cannot log in anymore when your username includes a point (example.name)

**Comment by lifenautjoe** [comment]
Yep! In progress here https://github.com/OpenbookOrg/openbook-app/issues/99
